Processen/CR-PR-API-Beheer.md

```diff
@@ -14,6 +14,32 @@ Het onderstaande diagram beschrijft hoe er wordt om gegaan met issues die in het
 
 De stappen in het Change- en Problem Management proces van API-Beheer zijn genummerd met een uniek nummer en hieronder in meer detail beschreven.
```

Comment:
iets verder terug staat "als onderwerp een API standaard of Referentie Implementatie" --> referentie-implementatie is onderdeel van de standaard. dus het woord standaard is voldoende.

Reply:
Heb ik aangepast.